src/main/java/io/stargate/sgv2/jsonapi/service/bridge/executor/NamespaceCache.java

```diff
@@ -41,14 +42,24 @@ protected Uni<CollectionSettings> getCollectionProperties(String collectionName)
           .transformToUni(
               (result, error) -> {
                 if (null != error) {
+                  // collection does not exist
+                  if (error instanceof RuntimeException rte
```

Comment:
Is there no `cause` to check for type or something, instead of relying exception message (which may change)?